test-app/build-tools/android-metadata-generator/src/src/com/telerik/metadata/Generator.java

```diff
@@ -72,6 +75,9 @@ private static void enableFlaggedFeatures(String[] args) {
                 AnalyticsConfiguration.enableAnalytics(filePath);
             } else if (VERBOSE_FLAG_NAME.equals(arg)) {
                 MetadataFilterConsoleLogger.INSTANCE.setEnabled(true);
+            } else if (SKIP_FLAG_NAME.equals(arg)) {
+                System.out.println("Exiting");
```

Comment:
```suggestion
                System.out.println("Skipping metadata generation: skip flag used");
```

Reply:
Yeah this was really used as a internal item flag so that it would destroy my metadata generated by another NS instance as the runtime app needs some significant work to make it easy to run example apps in it.    But yeah adding a better message probably makes since.  Updated.